src/main/webapp/resources/js/pages/projects/samples/components/SamplesTable.jsx

```diff
@@ -43,33 +43,31 @@ export function SamplesTable() {
   } = useSelector((state) => state.samples);
 
   /**
-   * Fetch the current state of the table.  Will refetch whenever one of the
+   * Fetch the current state of the table.
+   * Re-fetch whenever one of the
    * table options (filter, sort, or pagination) changes.
    */
-  const {
-    data: { content: samples, total } = {},
-    isFetching,
-  } = useListSamplesQuery(options, {
-    refetchOnMountOrArgChange: true,
-  });
+  const { data: { content: samples, total } = {}, isFetching } =
+    useListSamplesQuery(options, {
+      refetchOnMountOrArgChange: true,
+    });
 
   /**
    * Fetch projects that have been associated with this project.
    * Request formats them into a format that can be consumed by the
    * project column filter.
    */
-  const { data: associatedProjects } = useListAssociatedProjectsQuery(
-    projectId
-  );
+  const { data: associatedProjects } =
+    useListAssociatedProjectsQuery(projectId);
 
   /**
    * Handle row selection change event
    * @param event
    * @param sample
    */
   const onRowSelectionChange = (event, sample) => {
-    const selected = event.target.checked;
-    if (selected) {
+    console.log(selected);
```

Comment:
Remove logging statement

Reply:
Removed in d73f8bb385198793f63345d399f52a4226e6be79

Reply:
The logging statement is still in this file